src/content-handlers/iiif/modules/uv-shared-module/css/mixins.less

```diff
@@ -648,6 +648,14 @@
   }
 }
 
+// Loading
+// -------------------------
+.loading(@img) {
```

Comment:
This is already defined in [mixins-extended.less](https://github.com/UniversalViewer/universalviewer/blob/3ee2eec5de1504dc07c7265f5415f8081ccceb14/src/content-handlers/iiif/modules/uv-shared-module/css/mixins-extended.less#L41-L45). Do we also need it here, or can we leverage that existing definition?

Reply:
Ah!   Missed it in the extended file.  Yes, was able to remove this and it's falling back on the expected styles.